spec/javascript/testHelper.js

```diff
@@ -1,6 +1,7 @@
 import { shallow, mount } from 'enzyme';
 import jasmineEnzyme from 'jasmine-enzyme';
 import React from 'react';
+import { BrowserRouter, Router, Link, Route } from 'react-router-dom';
```

Comment:
I don't think this line is actually doing anything for you -- you're not assigning these as globals below. Please delete this line (or let me know if I'm mistaken!)